src/group_theory/nilpotent.lean

```diff
@@ -308,6 +313,29 @@ begin
     use [lower_central_series G, lower_central_series_is_descending_central_series, h] },
 end
 
+lemma lower_central_series_subgroup_le_lower_central_series_group (H : subgroup G) (n : ℕ) :
+  (lower_central_series H n).map H.subtype ≤ lower_central_series G n :=
+begin
+  induction n with d hd,
+  { simp },
+  { rw [lower_central_series_succ, lower_central_series_succ, monoid_hom.map_closure],
+    apply subgroup.closure_mono,
+    rintros x1 ⟨x2, ⟨x3, hx3, x4, hx4, rfl⟩, rfl⟩,
+    exact ⟨x3, (hd (mem_map.mpr ⟨x3, hx3, rfl⟩)), x4, by simp⟩ }
+end
+
+lemma nilpotent_to_nilpotent_subgroups (G : Type*) [group G] (H : subgroup G) :
```

Comment:
This should be an instance, and the is_nilpotent G argument should be an instance argument